src/App.js

```diff
@@ -8,10 +8,10 @@ import './App.css'
 
 export default function App() {
     const [todoItems, setTodoItems] = useState([])
-    const [editId, setEditId] = useState()
+    const [itemToEdit, setItemToEdit] = useState(null)
 
     function editTodo(id) {
-        setEditId(id)
+        setItemToEdit(id)
```

Comment:
`itemToEdit`, not `id`

You should use the id passed to find the itemToEdit.

Reply:
Here, I already passed in id as an argument to the handleEdit prop in App.js

Reply:
So I want to set itemToEdit to be equal to the id